lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -691,10 +697,10 @@ namespace Js
         __finally
         {
             // 0xE06D7363 is C++ exception code
-            if (exceptionCode != 0 && !IsDebuggerPresent() && exceptionCode != 0xE06D7363 && exceptionAction != EXCEPTION_CONTINUE_EXECUTION)
+            if (AbnormalTermination() && exceptionCode != 0 && !IsDebuggerPresent() && exceptionCode != 0xE06D7363)
```

Comment:
It is actually very cheap.  It is just a local variable managed by the compiler.

---
In reply to: [100167801](https://github.com/Microsoft/ChakraCore/pull/2493#discussion_r100167801) [](ancestors = 100167801)